tests/unit_tests/data_steward/tools/run_deid_test.py

```diff
@@ -16,11 +33,79 @@ def setUpClass(cls):
         print('**************************************************************')
 
     def setUp(self):
-        pass
+        # input parameters expected by the class
+        self.input_dataset = 'foo_input'
+        self.private_key = 'fake/SA/file/path.json'
+        self.output_dataset = 'foo_output_deid'
+        self.action = 'debug'
+        self.skip_tables = 'foo_table'
+        self.tablename = 'bar_table'
+
+        self.correct_parameter_list = [
+            '--idataset', self.input_dataset, '--private_key', self.private_key,
+            '--odataset', self.output_dataset, '--action', self.action,
+            '--skip-tables', self.skip_tables, '--tables', self.tablename
+        ]
+
+        self.incorrect_parameter_list = [
+            '--idataset',
+            self.input_dataset,
+            '--private_key',
+            self.private_key,
+            '--action',
+            self.action,
+            '--skip_tables',
+            self.skip_tables,
+            '--tables',
+            self.tablename,
+        ]
 
     def tearDown(self):
         pass
 
+    def test_parse_args(self):
+        # Tests if incorrect parameters are given
+        self.assertRaises(SystemExit, parse_args, self.incorrect_parameter_list)
+
```

Comment:
```
        self.incorrect_parameter_list.extend(['--odataset', 'random_deid_tag'])
        self.assertRaises(SystemExit, run_deid.parse_args, self.incorrect_parameter_list)
```
Add this test.  This test will fail because the SystemExit is not raised.

Reply:
I added this to parser_test.py as well